testbed/tests/widgets/test_slider.py

```diff
@@ -90,16 +90,16 @@ async def test_change(widget, probe, on_change):
         widget.max = scale
         for position in POSITIONS:
             on_change.reset_mock()
-            probe.change(position)
+            await probe.change(position)
+            await probe.redraw("Slider scale should be %s" % scale)
             assert widget.value == approx(position * scale, abs=(ACCURACY * scale))
             on_change.assert_called_once_with(widget)
-            await probe.redraw("Slider scale should be %s" % scale)
 
     on_change.reset_mock()
     widget.on_change = None
-    probe.change(0.42)
-    on_change.assert_not_called()
+    await probe.change(0.42)
```

Comment:
This was the bug - it was causing a [warning about an unawaited co-routine](https://github.com/beeware/toga/actions/runs/5372567106/jobs/9746615355?pr=2011#step:5:547)